GameData/RP-0/Contracts/Moon Exploration/Lunar Orbiter.cfg

```diff
@@ -43,16 +43,50 @@ CONTRACT_TYPE
 		type = CompleteContract
 		contractType = first_OrbitScience  // If Pioneer 0 would have worked
 	}
-	
+
+	DATA
+	{
+		type = List<int>
+		advances = [18750, 4500]
+		rewards = [18750, 14000]
+	}
+
+	DATA 
+	{
+		type = int
+		advance = @advances.ElementAt($first_MoonOrbitUncrewed_Count)
+		reward = @rewards.ElementAt($first_MoonOrbitUncrewed_Count)
+	}
+
+	DATA
+	{
+		type = List<float>
+		minPeList = [20000, 20000]
+		maxApList = [3000000, 2000000]
```

Comment:
These two values for the Apoapsis are kind of a meaningless progression. The dv difference is negligible. I would suggest to go for 5,000 km in the first and 500 km in the second, at least.

Reply:
@siimav Your thoughts on this would be nice

Reply:
I agree with @leudaimon that 3Mm -> 2Mm makes almost no difference. 5Mm is incidentally what we had before Pap went in and made the contract harder. Setting the Ap requirement of second completion to <=500km would almost completely prevent unguided shenanigans here if that's what we want.